### PR TITLE
Added a new feature (James)

### PR DESCRIPTION
**Pull request by NoodlesCupCake**
This feature is for adding an autocomplete in the search box.

### CODE REVIEW

File: lab-exercise-1/JirayuFeature.java [hunk 1]
```diff
@@ -1,7 +1,5 @@
   public class JirayuFeature {
       public void feature() {
           // this is a new feature
```
Comment (NoodlesCupCake) [inline lab-exercise-1/JirayuFeature.java:3]: Improve this thing please.

File: lab-exercise-1/JirayuFeature.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+          int x = 0;
+          x = x+1;
```
Comment (NoodlesCupCake) [inline lab-exercise-1/JirayuFeature.java:5]: good job, James.

**Review by ChanAndKlee — COMMENTED** [review]
Please check here

File: lab-exercise-1/JirayuFeature.java (hunk 1 above)
Comment (KRoos555) [reply lab-exercise-1/JirayuFeature.java:3]: Please improve this

File: lab-exercise-1/JirayuFeature.java (hunk 2 above)
Comment (Vvidsky) [inline lab-exercise-1/JirayuFeature.java:5]: Good job, James.

**Review by Vvidsky — APPROVED** [review]

File: lab-exercise-1/JirayuFeature.java (hunk 1 above)
Comment (Jack5hj) [reply lab-exercise-1/JirayuFeature.java:3]: please improve this

Comment (Pontuspts) [reply lab-exercise-1/JirayuFeature.java:3]: Improve this

File: lab-exercise-1/JirayuFeature.java (hunk 2 above)
Comment (Pontuspts) [reply lab-exercise-1/JirayuFeature.java:5]: good job man

Comment (ChanAndKlee) [reply lab-exercise-1/JirayuFeature.java:5]: Good!

**Pull request merged**